ember-cli-flash/package.json

```diff
@@ -49,7 +49,8 @@
     "test": "echo 'A v2 addon does not have tests, run tests in test-app'"
   },
   "dependencies": {
-    "@embroider/addon-shim": "^1.0.0"
+    "@embroider/addon-shim": "^1.0.0",
+    "@embroider/macros": "^1.13.2"
```

Comment:
I'm not sure this should be a peer since it's not in the app blueprint.

I get confused about some of these build packages, though 😅  Are we missing other dependencies here like `ember-auto-import`?

Reply:
Auto-import is only needed for v1 addons.

I think peer is correct, because it communicates that the consumer needs to do something

Reply:
New PR for this #395